conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+azure:
+  store_build_artifacts: true
```

Comment:
Guessing this is just for testing

Whenever testing is done, would recommend dropping this and re-rendering

```suggestion
```